src/NServiceBus.AzureStoragePersistence.SagaDeduplicator.Tests/ProgramTests.cs

```diff
@@ -32,7 +33,7 @@ public ProgramTests()
         [SetUp]
         public void SetUp()
         {
-            cloudTable = cloudTableClient.GetTableReference("Deduplicator" + Guid.NewGuid().ToString().Replace("-", ""));
+            cloudTable = cloudTableClient.GetTableReference(typeof(TwoInstanceSagaState).Name);
```

Comment:
It does not use the `AzureSagaPersister` but that's the way `AzureSagaPersister` obtains the name of the table.